server/src/tree-sitter/analyzer.ts

```diff
@@ -29,6 +29,7 @@ import { bitBakeProjectScannerClient } from '../BitbakeProjectScannerClient'
 import { bitBakeDocScanner } from '../BitBakeDocScanner'
 
 export interface AnalyzedDocument {
+  version: number // TextDocument is mutable and its version updates as the document updates
```

Comment:
This has the side effect of not having embedded language features when a document is just being opened but not modified. But OK for now.